apps/api/src/form/form.controller.ts

```diff
@@ -43,4 +50,39 @@ export class FormController {
   async getFormById(@Param('id') id: string): Promise<FormEntity> {
     return await this.formService.getFormById(id);
   }
+
+  @ApiOperation({
+    summary: 'Export Data',
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @ApiResponse({ status: HttpStatus.CREATED, type: EmptyResponse })
+  @HttpCode(HttpStatus.CREATED)
+  @Get('/export/:passCode')
```

Comment:
I think passcode should be in a header rather than a query param, no?

Reply:
No. the end users cant make a API call by setting headers. We also cannot share a url if we do this. That is the reason.

Reply:
Who are the end users of this endpoint? I assumed it was just us

I can't logically think of a reason I prefer a header (other than tracking in a browser's history), if you have a preference for a query param then that's alright

Reply:
Assumption - 

MVP: 
we share the URL which has the passcode to MoH user. 
MoH User downloads a copy whenever they want 

> This way, there needs to be no back and forth and they can generate as many as often as required 

Later: 

Have a login + UI + etc.